lib/slither/column.rb

```diff
@@ -56,7 +56,7 @@ def formatter
       end
 
       def aligner
-        @alignment == :left ? '-' : ''
+        @alignment == :left ? '-'.freeze : ''.freeze
```

Comment:
How would `.freeze` will use less memory? Do you have some weblink for explanation? I'm curious.

Reply:
When you freeze a string it will behave like a symbol or constant. It will have an own memory, ruby does not have to allocate memory anymore.

Reply:
https://freelancing-gods.com/2017/07/27/an-introduction-to-frozen-string-literals.html